### PR TITLE
Change the category of messages

### PR DESCRIPTION
**Pull request by jan-cerny**
These messages are disturibing, moreover there is no way
how a user can mitigate them.

### CODE REVIEW

File: src/OVAL/probes/unix/linux/rpminfo.c [hunk 1]
```diff
@@ -143,7 +143,7 @@ static void pkgh2rep (Header h, struct rpminfo_rep *r)
 
 	if (regexec(&g_keyid_regex, str, 1, keyid_match, 0) != 0) {
 		sid = NULL;
-		dW("Failed to extract the Key ID value: regex=\"%s\", string=\"%s\"",
+		dI("Failed to extract the Key ID value: regex=\"%s\", string=\"%s\"",
```
Comment (yuumasato) [inline src/OVAL/probes/unix/linux/rpminfo.c]: This looks like it should be a developer level warning, it has regular expression and a string.
It is not clear to me what is criteria.

Comment (jan-cerny) [inline src/OVAL/probes/unix/linux/rpminfo.c]: @yuumasato yes, you're true

Comment (jan-cerny) [inline src/OVAL/probes/unix/linux/rpminfo.c]: changed

**Comment by jan-cerny** [comment]
Also can fix #716

**Comment by yuumasato** [comment]
Ack, thank you.

**Pull request merged**